modules/test-tools/src/main/kotlin/jp/co/soramitsu/iroha2/testengine/IrohaContainer.kt

```diff
@@ -116,7 +119,7 @@ open class IrohaContainer : GenericContainer<IrohaContainer> {
     override fun start() {
         logger().debug("Starting Iroha container")
         if (logger().isDebugEnabled) {
-            val genesisAsJson = config.genesis.asJson()
+            val genesisAsJson = config.genesis?.asJson()
```

Comment:
actually it's going to use `Genesis.getEmpty()` right? 
should be smth like `<genesisPath's value> ?: config.genesis?.asJson()`